community/sig-spec/README.md

```diff
@@ -0,0 +1,24 @@
+# SIG Spec
+
+A Special Interest Group (SIG) concerning the official SPIFFE specifications.
+
+### Meetings:
+* Date/Time: TBD
+* [Meetings Notes](https://docs.google.com/document/d/1p2BbRWN_7z6nkBMj-h1mAJAJxxKqNeFiV4IplZ_wU4c)
+* Calendar ICS (TBD) (See this [Google support page](https://support.google.com/calendar/answer/37100?co=GENIE.Platform%3DDesktop&hl=en) to learn more about importing .ics files)
```

Comment:
@suniljames Are we still creating calendars for each SIG, or have we moved to invitations? I couldn't find a calendar for SIG-Spec